literature/literature.bib

```diff
@@ -469,6 +469,33 @@ @article{HGFB:SCP24
 	ektags    = {complexity},
 }
 
+@article{MCS+:TSE24,
+	author    = {Mahmood, Wardah and Çalıklı, Gül and Strüber, Daniel and Lämmel, Ralf and Mukelabai, Mukelabai and Berger, Thorsten},
+	title     = {{Virtual Platform: Effective and Seamless Variability Management for Software Systems}},
+	journal   = tse,
+	volume    = {50},
+	number    = {11},
+	pages     = {2753--2785},
+	doi       = {10.1109/TSE.2024.3406224},
+	publisher = ieee,
+	year      = 2024,
+}
```

Comment:
I think this is a journal extension of an ICSE paper. We should probably mark the ICSE entry as subsumed then.